### ISSUE TITLE
PrefixAllGlobals: false negatives when checking group constant declarations

### ISSUE THREAD

**Issue by rodrigoprimo** [WordPress/WordPress-Coding-Standards#2560]
## Bug Description

The sniff `WordPress.NamingConventions.PrefixAllGlobals` does not take into consideration that multiple constants can be declared using a single `const` keyword. When there is more than one constant declaration, the sniff will check only the name of the first constant, potentially leading to false negatives.

## Minimal Code Snippet

The issue happens when running this command:
```bash
vendor/bin/phpcs -s --standard=WordPress --sniffs=WordPress.NamingConventions.PrefixAllGlobals test.php
```

... over a file containing this code:
```php
<?php

// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] my_plugin

const MY_PLUGIN_CONSTANT = 'value', ANOTHER_CONSTANT = 'another_value';
```

I would expect to see a `WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound` error for the `ANOTHER_CONSTANT` constant, but no error is reported.

## Error Code
`WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound`

## Environment

| Question                 | Answer
| ------------------------ | -------
| PHP version              | 8.3.22
| PHP_CodeSniffer version  | 3.13.2
| WordPressCS version      | `develop`
| PHPCSUtils version       | 1.1.0
| PHPCSExtra version       | 1.4.0
| WordPressCS install type | git clone
| IDE (if relevant)        | _N/A_

## Additional Context (optional)

I believe the problem is related to the code below when the sniff gets the name of the constant without considering that there might be more than one constant declaration after `T_CONST`:

https://github.com/WordPress/WordPress-Coding-Standards/blob/4d0160f32f8537f3cdf2e301433cb15641083963/WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php#L714-L733

## Tested Against `develop` Branch?
- [x] I have verified the issue still exists in the `develop` branch of WordPressCS.
